catalogue/engage.py

```diff
@@ -55,7 +55,11 @@ def git_query(repo_path, catalogue_dir, commit_changes=False):
             user_choice = input().strip().lower()
             if user_choice == "y" or user_choice == "yes":
                 timestamp = create_timestamp()
-                new_branch = repo.create_head("catalogue-" + timestamp)
+                try:
+                    new_branch = repo.create_head("catalogue-" + timestamp)
+                except BadName as e:
```

Comment:
You don't actually use `e` anywhere here -- did you mean to use it in the error message? I think it's fine the way it is, so I think you can just do `except BadName:`